src/App.tsx

```diff
@@ -17,62 +18,72 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  NONE = 'none',
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
 }
 
-export const App: React.FC = () => {
-  const [reversed, setReversed] = useState(false);
-  const [sortType, setSortType] = useState(SortType.NONE);
-
-  function getReorderedGoods(goods: string[]) {
-    const visibleGoods = [...goods];
-
-    switch (sortType) {
-      case SortType.ALPHABET:
-        visibleGoods.sort((firstGood, secondGood) => (
-          firstGood.localeCompare(secondGood)
-        ));
-        break;
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
 
-      case SortType.LENGTH:
-        visibleGoods.sort((firstGood, secondGood) => (
-          firstGood.length - secondGood.length
-        ));
-        break;
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+): string[] {
+  const visibleGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPHABET:
+      visibleGoods.sort((firstGood, secondGood) => (
+        firstGood.localeCompare(secondGood)
+      ));
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((firstGood, secondGood) => (
+        firstGood.length - secondGood.length
+      ));
+      break;
+
+    default:
+      break;
+  }
 
-      case SortType.NONE:
-      default:
-        break;
-    }
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
 
-    if (reversed) {
-      visibleGoods.reverse();
-    }
+  return visibleGoods;
+}
 
-    return visibleGoods;
-  }
+export const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState(false);
+  const [sortType, setSortType] = useState(SortType.NONE);
 
-  const handleAlphabet = () => {
+  const SortByAlphabet = () => {
```

Comment:
```suggestion
  const sortByAlphabet = () => {
```

Use camelCase for methods